### ISSUE TITLE
Demo for the README

### ISSUE THREAD

**Issue by aendrew** [CBiX/svgToPdf.js#13]
This project looks great, but I can't really adequately evaluate whether it would be good for my project without having a way to demo its capabilities. It'd be really cool to have a Plunker that loads in an SVG document and then renders to PDF on button click — or better yet, a GitHub Pages page that allows users to upload (via Blob interface)/paste a SVG document into a textbox, with svgToPdf.js converting it to a downloaded PDF. Happy to do a pull request if that'd be helpful!

**Comment by CBiX** [comment]
Good idea, I could imagine having a website with 3 areas: SVG input (upload/text area), SVG input preview, output preview SVG (using the removeInvalid option) or the output PDF, respectively.

**Comment by aendrew** [comment]
@CBiX Cool, have some spare time tomorrow, will see what I can do!

**Comment by CBiX** [comment]
@aendrew Sounds great! Take a look at this code, it's were I originally used it: http://sourceforge.net/p/ocsvg/code/ci/master/tree/js/ocsvgEditor.js#l194

In the meantime (since 2012) some things changed but you'll figure that out ;)

**Comment by aendrew** [comment]
@CBiX I tried to do this earlier today and had a great deal of difficulty getting svgToPdf.js to render anything; additionally, TIL there's really no way of POSTing a file back to the web-browser, without using a webserver. Thus the best option would probably be to just use a textbox, I think?
